data/json/monsters/zed_radiation.json

```diff
@@ -2,8 +2,8 @@
   {
     "id": "mon_charred_nightmare",
     "type": "MONSTER",
-    "name": { "str": "charred nightmare" },
-    "description": "Intense radiation has spurred a unique form of necrosis and regeneration, it is impossible to tell if this creature was ever human.",
+    "name": { "str": "atomic nightmare" },
```

Comment:
I don't really see the point for the name change.

Reply:
Its a flavour thing more than anything, its meant to be more fitting to its description, and where its found, compared to the implied burned status, if y'all want i can switch it back no sweat

Reply:
I have reverted the change.